src/controllers/users.js

```diff
@@ -52,22 +44,17 @@ export async function getUser(req, res) {
     if (!req.get("Authorization")) {
       throw new Error("Authorization header empty");
     }
-    const token = await req.get("Authorization").split(" ").pop();
-
-    const decode = jwt.verify(token, process.env.PUBLIC_KEY, { algorithms: 'RS256' })
-
-    const user = await UserModel.findByPk(decode.email)
-
-    const responseData = userResponse(user.get())
-
-    return res.status(200).json(responseData)
-
+    const token = getToken(req.get("Authorization"));
+    const decode = verifyToken(token);
```

Comment:
something to think about for the future, there should be an authorization middleware that handles token checks.

Reply:
I'll create a middleware for authorization routes

Reply:
I'll make the updates on the next pull request